### PR TITLE
Improve compose box.

### PR DESCRIPTION
**Pull request by jainkuniya**
part of #2039

### CODE REVIEW

File: src/autoComplete/AutoCompleteViewWrapper.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { StyleSheet, View } from 'react-native';
 
 import type { InputSelectionType, Narrow } from '../types';
 import AutoCompleteView from './AutoCompleteView';
-import TopicAutocomplete from './TopicAutocomplete';
```
Comment (borisyankov) [inline src/autoComplete/AutoCompleteViewWrapper.js]: https://en.wikipedia.org/wiki/Autocomplete

Autocomplete is one word, thus the casing is correct!

Comment (jainkuniya) [reply src/autoComplete/AutoCompleteViewWrapper.js]: Done! refactored all to `autocomplete`.
Not sure then why `eslint` is not considering it.

**Comment by gnprice** [comment]
Awesome -- thanks @jainkuniya for the very cleanly separated commits, and the nice explanatory commit messages!

A few small things would make these commit messages even better (summarizing discussion we had in chat):
* Please line-wrap paragraphs to <= 75 columns wide -- this makes them read nicely in `git log`.
* Try prefixing each summary line with a word or two identifying the code it affects, then a colon. This helps out when scanning through a lot of changes in e.g. `gitk`.
* Summary lines can be as long as about 66-70 columns wide, so there's room for more info there; try squeezing in a bit of the "why" in addition to "how" and "what". (And then expanding on all of those at more length in the body of the commit message, as you do already in this version.)

An example of what the summary lines could look like following those latter two points:
```
ComposeBox: Factor out autocomplete as a separate component.
MultilineInput: Simplify by using rest/spread to pass properties through.
ComposeBox: Let height vary automatically to avoid lag.
ComposeBox: Add onLayout callback to adjust autocomplete.
ComposeBox: Tweak layout, and slightly refactor styles.
autocomplete: Consistently capitalize "autocomplete" as one word.
```

**Comment by borisyankov** [comment]
After your changes, the code looks reasonable and is a nice improvement on an important part of the app. Ready to merge.

**Pull request merged**